include/libkernelflinger/log.h

```diff
@@ -38,9 +38,15 @@
 #include <vars.h>
 
 EFI_STATUS log_flush_to_var(BOOLEAN nonvol);
+EFI_FILE * efi_log_file;
```

Comment:
**Coding style violation**: `*` is misplaced, it should be `EFI_FILE *efi_log_file;`

Same below.

Reply:
Already modified.